charts/kube-prometheus-stack/templates/prometheus/rules-1.14/prometheus.yaml

```diff
@@ -256,20 +256,20 @@ spec:
         summary: Prometheus is not ingesting samples.
       expr: |-
         (
-          rate(prometheus_tsdb_head_samples_appended_total{job="{{ $prometheusJob }}",namespace="{{ $namespace }}"}[5m]) <= 0
```

Comment:
Not intendet.

Reply:
Same, it's from upstream [pr 2344](https://github.com/prometheus-operator/kube-prometheus/pull/2344) and [rule diff](https://github.com/prometheus-operator/kube-prometheus/pull/2344/files#diff-2a066447e0230293dd113775548de681d94adcaeb1b32f8f9ab465411c8b6770R111)